morula.yml

```diff
@@ -1,5 +1,5 @@
 before-all: "exorelay-javascript"
 after-all: ""
 always: ""
-never: ""
+never: "bin"
```

Comment:
The correct thing is to add `bin` here, since there is nothing Morula should ever do here.